Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -110,11 +110,10 @@ public void GenerateFile_ValidFiles()
         // Arrange
         var testDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
 
-        TestUtils.CreateProjectInfoInSubDir(testDir, "withoutFiles", null, Guid.NewGuid(), ProjectType.Product, false, Path.Combine(testDir, "abc", "withoutfile.proj"), "UTF-8"); // not excluded
+        TestUtils.CreateProjectWithFiles(TestContext, "withoutFiles", testDir, createContentFiles: false);
```

Comment:
This creates the `withoutfile.proj` implicitly, which is needed after the new check.

Reply:
This does not read nicely. CreateProjectWithFiles has a "withoutFiles" parameter. I'm confused now.